float64 base
float64 shoulder
float64 elbow 
float64 gripper
---
float64 base
float64 shoulder
float64 elbow 
float64 gripper